src/components/PopupBase/__tests__/PopupBase.spec.tsx

```diff
@@ -14,28 +16,32 @@ describe('<PopupBase> that adds basic anchor/popup functionality to rendered com
     const popupRendererMock = ({ setPopupVisibility }) => (
         <PopoverDummy setPopupVisibility={setPopupVisibility} />
     );
-    let wrapper;
+    let view: RenderResult;
 
     describe('rendering', () => {
         beforeEach(() => {
-            wrapper = mount(
+            view = render(
                 <PopupBase anchorRenderer={anchorRendererMock} popupRenderer={popupRendererMock} />
             );
         });
 
         it('should render with minimal props correctly', () => {
-            expect(toJson(wrapper)).toMatchSnapshot();
-            expect(wrapper.find('Popover')).toHaveLength(0);
+            expect(view.container).toMatchSnapshot();
+            expect(screen.queryByRole('group')).not.toBeInTheDocument();
         });
-        it('should render popup when requested', () => {
+
+        it('should render popup when requested', async () => {
+            const user = userEvent.setup();
```

Comment:
you can do this in the beforeEach because it looks like it is being used at every test.

Reply:
I don't remember why but we should init this user in each test case. At least we do it for all tests

Reply:
Doing this in the beforeEach we also init this user in each test case, we just don't need to repeat the line of code in every test.

Reply:
:+1: